pkg/controller/controller.go

```diff
@@ -201,7 +206,22 @@ type requiredCapabilities struct {
 	clone    bool
 }
 
-// CSIProvisioner struct
+// NodeDeployment contains additional parameters for running external-provisioner alongside a
+// CSI driver on one or more nodes in the cluster.
+type NodeDeployment struct {
+	NodeName         string
```

Comment:
Can you add some comments on what each of these fields do?

Reply:
Done.